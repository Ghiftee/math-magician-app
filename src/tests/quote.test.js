import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../pages/quotes';

test('expect quote page to match working snapshot', () => {
  const quotePage = renderer.create(<Quote />).toJSON();
  expect(quotePage).toMatchSnapshot();
});
