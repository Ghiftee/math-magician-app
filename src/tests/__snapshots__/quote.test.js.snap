// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote page to match working snapshot 1`] = `
<div>
  <p
    style={
      Object {
        "fontSize": "22px",
        "marginTop": "50px",
        "padding": "30px",
      }
    }
  >
    Mathematics is not about numbers, equations, computations, or algorithms: It is about understanding. -- William Paul Thurston
  </p>
</div>
`;
